packages/hardhat-core/test/internal/hardhat-network/stack-traces/test-files/0_8/console-logs/error/assert/c.sol

```diff
@@ -9,4 +9,10 @@ contract C {
     assert(b);
   }
 
+  // test2 is needed as workaround for the source mappings issue
```

Comment:
So for now I've fixed the issue with the `console-logs::error::assert` test by adding another function in the contract which forces solc to leave jumps from the `console` contract back to the `C` contract (to differentiate between `test` and `test2` calls) and also to generate proper source mappings for those jumps.

With single `test` function I believe those jumps were "optimized" by inliner and source mappings were unusable because of that optimization.